react/src/components/ModelTryContentButton.tsx

```diff
@@ -236,33 +230,143 @@ const ModelTryContentButton: React.FC<ModelTryContentButtonProps> = ({
     return {
       serviceName: `${modelName}-${generateRandomString(4)}`,
       replicas: 1,
-      environments: modelCardMetadata?.environments || {
+      environments: {
         environment: '',
         version: '',
         image: null,
       },
-      runtimeVariant: modelCardMetadata?.runtimeVariant || 'custom',
+      runtimeVariant: 'vllm',
       cluster_size: 1,
       cluster_mode: 'single-node',
       openToPublic: true,
       resourceGroup: currentResourceGroupByProject as string,
-      resource: modelCardMetadata?.resource || {
+      resource: {
         cpu: 4,
         mem: '32g',
         accelerator: 10,
         acceleratorType: 'cuda.shares',
         shmem: '1g',
       },
-      vFolderID: vfolderID, // TODO: add cloned folder result
+      vFolderID: vfolderID,
       modelMountDestination: '/models',
       modelDefinitionPath: '',
       mount_id_map: {},
-      envvars: modelCardMetadata?.envvars || [],
+      envvars: [],
       enabledAutomaticShmem: false,
     };
   };
 
-  const cloneOrCreateModelService = (runtimeVariant: string) => {
+  // Check if folder has required definition files
+  const checkFolderDefinitionFiles = async (vfolderId: string) => {
+    try {
+      const filesResponse = await baiClient.vfolder.list_files('', vfolderId);
+      const files = filesResponse?.items || [];
+      const hasModelDefinition = _.some(
+        files,
+        (file: { name?: string }) => file?.name === 'model-definition.yaml',
+      );
+      const hasServiceDefinition = _.some(
+        files,
+        (file: { name?: string }) => file?.name === 'service-definition.toml',
+      );
+
+      return {
+        hasModelDefinition,
+        hasServiceDefinition,
+        files,
+      };
+    } catch (error) {
+      console.error('Error checking folder files:', error);
+      // Provide more specific error information
+      const errorMessage =
+        error instanceof Error ? error.message : 'Unknown error occurred';
+      console.error('Detailed error:', errorMessage);
+      return {
+        hasModelDefinition: false,
+        hasServiceDefinition: false,
+        files: [],
+        error: errorMessage,
+      };
+    }
+  };
+
+  // Helper function to create service notification
+  const createServiceNotification = (
+    result: ServiceResult,
+    modelId: string,
+  ) => {
+    upsertNotification({
+      key: result?.endpoint_id,
+      open: true,
+      message: t('modelService.StartingModelService'),
+      duration: 0,
+      backgroundTask: {
```

Comment:
The message field of an upsertNotification is used as the Title of the notification when used at the top, and as the description when used within a backgroundTask.